tests/helpers/non_sklearn_metrics.py

```diff
@@ -0,0 +1,52 @@
+"""File for non sklearn metrics that are to be used for reference for tests"""
+
```

Comment:
for the function typing above
```suggestion
from typing import Optional
```